browser/src/global.d.ts

```diff
@@ -131,6 +131,7 @@ interface Window {
 	// app defined in: js/bundle.js
 	app: {
 		colorPalettes: any; // TODO declare according to Widget.ColorPicker.ts
+		colorNames: any; // TODO declare according to Widget.ColorPicker.ts
```

Comment:
we could add some type here - in followup

Reply:
I proposed in the core to change from simple string into struct for every entry for future extensibility